internal/pkg/mining/scheduler_test.go

```diff
@@ -17,33 +17,34 @@ import (
 	"github.com/filecoin-project/go-filecoin/internal/pkg/types"
 )
 
+const epochDuration = 1 * time.Second
```

Comment:
With a fake clock we should be able to use long durations here but still ahve the test be fast. If that's not true, please mark as an integration test, and file an issue or follow up to make it true.